test/foundry/BlastPoints.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {OwnableTwoSteps} from "../../contracts/OwnableTwoSteps.sol";
+import {BlastPoints} from "../../contracts/BlastPoints.sol";
+import {TestHelpers} from "./utils/TestHelpers.sol";
+
+import {MockBlastPoints} from "../mock/MockBlastPoints.sol";
+
+contract BlastPoints_Test is TestHelpers {
+    MockBlastPoints private mockBlastPoints;
+    BlastPoints private blastPoints;
+
+    address public operator = address(420);
+    address private constant TREASURY = address(69420);
```

Comment:
this isn't used

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/68bb39c481845cf5d2ec1da288916bba66c0a300